test/xbgpu/test_engine.py

```diff
@@ -553,9 +659,9 @@ def heap_factory(batch_index: int) -> list[spead2.send.HeapReference]:
         # Depending on the `missing_antenna` parameter, the full accumulations
         # will either be all complete or incomplete.
         if missing_antenna is not None:
-            expected_sensor_updates += [(False, aiokatcp.Sensor.Status.ERROR)] * n_full_accumulations
+            expected_sensor_updates += [(False, aiokatcp.Sensor.Status.ERROR)] * (max(n_accumulations_completed) - 1)
```

Comment:
At present, actual_sensor_updates is collecting from output 0, so this should probably use `n_accumulations_completed[0]` rather than the maximum.